template/src/pages/index/UnAuthApp.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { Redirect, Switch, Route } from 'react-router-dom';
+import routes from '../../routes/unAuthRoutes';
+
+function UnAuthApp() {
+  return (
+    <Switch>
+      {routes.map((r) => (
+        <Route {...r} key={r?.path as string} />
+      ))}
```

Comment:
Particularly, I don't like this approach. Anyway, I change after the boilerplate is generated 😅.

Reply:
same. I prefer explicit and declarative props.

Reply:
I agree with @phenrique7 on this. I don't see any benefits on using an array of routes, especially for unauthenticated routes. An array of routes can be useful if you're going to reuse in a sidebar, or if you're going to filter the array based on user permissions. But, in this case, you're just switching from writing the routes here to writing them elsewhere.

Reply:
Also, my review on this is the same for [authenticated routes](https://github.com/t10d/cra-template-kingdom/blob/main/template/src/pages/index/AuthApp.tsx#L9-L11).